### PR TITLE
Add force-reboot and drain timeouts to chart config and ds

### PR DESCRIPTION
**Pull request by cnmcavoy**
Followup from https://github.com/weaveworks/kured/pull/341, exposes all of the new cli flags to the helm chart.

### CODE REVIEW

**Comment by anthr76** [comment]
Would it be possible to add reboot command as well or is another PR warranted?

**Comment by evrardjp** [comment]
@anthr76 the reboot command is already flexible, in our main branch. We are just waiting for the next kubernetes kind nodes to release our recent work!

File: cmd/kured/main.go
Comment (evrardjp) [inline cmd/kured/main.go:99]: if you would like this clarification to be included in the next release (1.7.0), it's probably better to add it into another PR.

Comment (cnmcavoy) [reply cmd/kured/main.go:99]: I'm not overly concerned, I just noticed it was slightly incorrect when doing the helm chart changes and wanted to correct it.

**Comment by bstrdsmkr** [comment]
> @anthr76 the reboot command is already flexible, in our main branch. We are just waiting for the next kubernetes kind nodes to release our recent work!

I'm not sure I understand your response here -- I believe @anthr76 is asking to have the `--reboot-command` included as an explicit option in the chart, even though it (as are these) is already settable via `extraArgs`. I didn't see an explicit option in the chart in the main branch. Did I just miss it?

**Comment by anthr76** [comment]
@bstrdsmkr Is indeed correct.. Didn't see that `extraArgs` bit before so not totally necessary. Just a nice to have at this point.

**Comment by evrardjp** [comment]
You're right, it seems like this slipped through the cracks. PRs welcome!

**Comment by anthr76** [comment]
I'll be happy to. Any idea on when this get's merged?

**Comment by evrardjp** [comment]
> 
> 
> as a sanity check, is there a generic way to convert between the flag name and a yaml safe name? That way you could generate them from the yaml values instead of hard coding each one

That would be amazing, but I am not sure we have that. @ckotzbauer what do you think of the idea? We could maybe write a utility function that does that in the helm chart, and leverage it? Or is there any other way?

**Comment by evrardjp** [comment]
@cnmcavoy this will need a rebase, can you tackle it? That would be lovely.

**Comment by ckotzbauer** [comment]
@evrardjp @bstrdsmkr I think the "kebabcase" function could help us: https://helm.sh/docs/chart_template_guide/function_list/#kebabcase

**Comment by EduardFazliev** [comment]
Is there any chance that this will be merged some day? Is there a way to help to speed up the process?

**Comment by ckotzbauer** [comment]
@cnmcavoy Can you rebase this again please?
I think this is should be ready to merge in general, if there are no other objections. I would be ok to include the clarification in `main.go`
/cc @evrardjp

**Comment by cnmcavoy** [comment]
rebased.

**Review by ckotzbauer — APPROVED** [review]

**Pull request merged**
